install/kubernetes/templates/addons/grafana-config/custom.ini

```diff
@@ -0,0 +1,9 @@
+[auth]
+disable_login_form = true
```

Comment:
aside: we should probably revisit this at some point, especially as these start to get exposed via ingress.

Reply:
Agreed. Until then, I like having this as a configmap as it is easier for users to update/customize from a local file/directory.